index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <header>
+            <nav class="nav_container">
+                <div class="logo">
+                     <img src="images/MyLogo.png" alt="Logo">
+                </div>
+     
+                <div class="toggle">
+                    <img src="images/Union.png" alt="Logo">
+                </div>
+             </nav>
+        </header>
+
+        <main>
+            <div class="main_container">
+               <div class="box1">
+                   I'm Alex Glad to see you!
+               </div>
```

Comment:
- [ ] Please consider using `h1` tag here instead of a `div` tag as this is the Header text that a user's eyes should see first when they enter your page and also to improve the semantic structure of your page. See my suggestion below:

```suggestion
               <h1 class="box1">
                   I'm Alex Glad to see you!
               </h1>
```